src/modules/tests/tests.effects.ts

```diff
@@ -36,4 +40,79 @@ export class TestsEffects {
     )),
   );
 
+  @Effect()
+  startPracticeTestEffect$ = this.actions$.pipe(
+    ofType(testActions.START_PRACTICE_TEST),
+    switchMap(() => {
+      const slot = {
+        slotData: {
+          ...practiceSlot,
+        },
+      };
+
+      return [
+        new PopulateApplicationReference(slot.slotData.booking.application),
```

Comment:
Could you avoid creating `slot` with `slotData` property and just navigate `practiceSlot` directly?

Reply:
Yes, nice spot. This was leftover from when this lived in the journal effects class and before I realised I didn't need to initialise quite as much state. `extractTestSlotAttributes` and `extractTestCentre` are also redundant now.